server/config.ts

```diff
@@ -80,7 +80,7 @@ export default {
   },
   apis: {
     communityApi: {
-      url: get('COMMUNITY_API_URL', 'http://localhost:8091', requiredInProduction),
+      url: get('COMMUNITY_API_URL', 'http://localhost:9092', requiredInProduction),
```

Comment:
is this intentional? this breaks local dev with docker-compose

if you want to alter this just for the tests you can alter test.env